src/lib/print-output.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-console */
+
+/**
+ * Helper for formatting and printing GitHub resources.
+ *
+ * @param {object} options
+ * @param {string} options.output - Output format. Supports: standard, json
+ * @param {object} options.resource
+ */
+module.exports = function printOutput({ output, resource }) {
+    if (output === 'json') {
+        console.log(JSON.stringify(resource, null, 2));
```

Comment:
You might want to remove the `JSON.stringify` spaces, so that it fits with the [JSON Lines](http://jsonlines.org/) spec. The use could then use it with `jq` to format if need be

```suggestion
        console.log(JSON.stringify(resource));
```

Reply:
Good point! Will change this.